### ISSUE TITLE
Session disappear after close and open tab, and only soft refresh can restore it

### ISSUE THREAD

**Issue by umberto10** [tinyzimmer/kvdi#24]
Hello again, 
I closed my browser tab after launching desktop image, then I closed the tab and reopen it. After login I couldn't attach to my session. I can see it from API as a session, but I there is no button I can click, to attach to this session. I saw [this](https://github.com/tinyzimmer/kvdi/pull/8) and [this](https://github.com/tinyzimmer/kvdi/issues/7), but I didn't find the answer. After few tries I checked browsers logs. 
This is after login: 
![image](https://user-images.githubusercontent.com/43953088/109137456-814c7b80-7759-11eb-8b76-631d436e199a.png)
This is after soft refresh (f5):
![image](https://user-images.githubusercontent.com/43953088/109137510-91645b00-7759-11eb-9624-afe7804feb1a.png)
After soft refresh it shows button and I can attach session. But I think it's some kind of bug :v
This is my config
```
vdi:
  spec:
    appNamespace: k8s-kvdi
    auth:
      tokenDuration: "8h"
      oidcAuth:
        preserveTokens: True
        # issuerURL: 
        issuerURL: 
        clientCredentialsSecret: "oidc-secret-kvdi"
        redirectURL: 
        groupScope: kvdi-policy
        adminGroups: [ "admin-user" ]
        allowNonGroupedReadOnly: True

**Comment by tinyzimmer** [comment]
Probably a bug somewhere in the UI. At least it works after a soft refresh. If I manage to make it happen to myself I'll see what I can do. Frankly, I'm getting to the point that in the near future I might be giving the UI a near-complete rewrite to flush out some of this stuff.

**Comment by tinyzimmer** [comment]
Closing this as I think it was fixed in `v0.3.2`

If it keeps happening, open a new issue.
